docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -104,6 +136,17 @@ Syntax: @helpurl _url_
 
 The provided _url_ is displayed in Excel.
 
+In the following example, the `helpurl` is www.contoso.com/help.
+
+```js
+/**
+ * A function which streams the temperature in a town you specify.
+ * @customfunction getTemp
```

Comment:
```suggestion
 * @customfunction getTemperature
```
'temp' sounds too close to 'temporary' so spelling out to avoid potential confusion